testing/bootstrap.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+#
+# This script is used to bootstrap our prow jobs.
```

Comment:
what's a prow job?

Reply:
https://github.com/kubernetes/test-infra/blob/master/prow/architecture.md

Reply:
haha, thanks.  I meant i wanted a comment from @Kashomon here ;)  I'm still looking for a one-sentence explanation of what it is and why i want it.

Reply:
How's this